docs/versions/v2.30.0/Release.md

```diff
@@ -79,5 +79,16 @@ When you trigger `mvn clean package install deploy` against base package, the bu
 #### Relevant Documentation:
 None
 
+### *Ability to cover cases when project.id is not set, but project.name is set when pushing Custom Resources*
+
+#### Previous Behavior
```

Comment:
Lets try to explain what the issue was and how it was resolved. 

Comments like: It didn't work  //// It works now is not really good.

In your case it could be:

#### Previous Behavior
Project id was being fetched from the configuration, which means in cases where the project id was not set, but project name was, there would be an error.

#### New Behavior
The project id is now fetched from the restClient, which checks if we have project id or name set and tries to find the correct id if the name is the only thing that is set